src/test/java/com/wikia/webdriver/common/templates/NewTestTemplateCore.java

```diff
@@ -117,9 +119,7 @@ protected void stopCustomBrowser(WebDriver customDriver) {
   protected DesiredCapabilities getCapsWithProxyServerSet(ProxyServer server) {
     capabilities = new DesiredCapabilities();
     try {
-      capabilities.setCapability(
-          CapabilityType.PROXY, server.seleniumProxy()
-      );
+      capabilities.setCapability(CapabilityType.PROXY, server.seleniumProxy());
     } catch (Exception e) {
       throw new RuntimeException(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)